spec/grape/extensions/param_builders/hash_spec.rb

```diff
@@ -79,5 +79,34 @@ def app
       expect(last_response.status).to eq(200)
       expect(last_response.body).to eq('["bar", nil]')
     end
+
+    it 'does not overwrite route_param with a regular param if they have same name' do
```

Comment:
Do we have similar specs without `Grape::Extensions::Hash::ParamBuilder` for the default behavior? Let's add otherwise.

Reply:
Sure, added similar tests for all param builders.